scripts/ci/build.sh

```diff
@@ -18,30 +18,27 @@ set -e
 readonly dir=$(dirname $0)
 readonly projectRoot=$dir/../..
 
+source ${projectRoot}/scripts/utils/maven.sh
+
 if [ -n "$GCLOUD_FILE" ]; then
-  source $dir/gcloud-init.sh
+  source ${dir}/gcloud-init.sh
 fi
 
-if [ -n "$DOCKER_NAMESPACE" ]; then
-  PROJECT_NAMESPACE="$DOCKER_NAMESPACE"
-else
-  PROJECT_NAMESPACE="gcr.io/$(gcloud info \
+pushd ${projectRoot}
+  export DOCKER_TAG_LONG=$(git rev-parse --short HEAD)
```

Comment:
Do you need to be at the git repository root to execute this command? I would have thought you can run this from anywhere within the repository.